leftwm-core/src/models/tag.rs

```diff
@@ -242,18 +242,30 @@ impl Tag {
     }
 
     pub fn update_windows(&self, windows: &mut Vec<Window>, workspace: &Workspace) {
-        if let Some(w) = windows
+        if let Some(window) = windows
             .iter_mut()
             .find(|w| w.has_tag(&self.id) && w.is_fullscreen())
         {
-            w.set_visible(true);
+            window.set_visible(true);
+            window.normal = workspace.xyhw;
+            let handle = window.handle;
+            windows
+                .iter_mut()
+                .filter(|w| {
+                    w.has_tag(&self.id)
+                        && w.transient.unwrap_or(WindowHandle::XlibHandle(0)) == handle
+                        && !w.is_unmanaged()
```

Comment:
I know it's kinda dumb, but I recently thought about adding a `window.is_managed()` method which just runs `!self.is_unmanaged()` internally. Because I found myself tripping up on the double negation of `!w.is_unmanaged()` more than once :laughing:

Reply:
We'd better address this minor QoL improvement in another PR though, I'd rather get this here merged as soon as possible :)